components/formats-api/src/loci/formats/WrappedReader.java

```diff
@@ -0,0 +1,473 @@
+/*
+ * #%L
+ * BSD implementations of Bio-Formats readers and writers
+ * %%
+ * Copyright (C) 2005 - 2017 Open Microscopy Environment:
```

Comment:
Can you fix the top-level the boilerplate to be`Top-level reader and writer APIs` and have `2018` as  the Copyright?